### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by Josh-McFarlin** [Josh-McFarlin/remix-image#4] [Ideas]
Have a new idea on how we can improve this package? List it here and we may add it!

**Comment by jfstn** [comment]
Hey @Josh-McFarlin 👋 
Is there a way to remove the server logs? Or at least being able to format them?

**Comment by christopherpickering** [comment]
Is there any way an alternate package to @next-boost/hybrid-disk-cache can be used? Even with the mem cache this package seems like it is required. I'm having a terrible time getting better-sqlite3 to install on my mac... first, my path had spaces in node-gyp died, and now there is some build error in c++ and the v8 engine. It would be cool to have something that doesn't need python and c++ to install. Do you have any ideas for a redis cache?

Ideally how I can get it running without the sqlite :)
